experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -1322,6 +1322,86 @@ describe("SharedTree", () => {
 	});
 
 	describe("Constraints", () => {
+		it("sequence field node exists constraint", () => {
```

Comment:
It doesn't need to be part of this PR, but I think we should port all (or almost all) of the constraint tests to `editing.spec.ts` and use the lighter testing patterns used in that file.